groundstation/backend_api/validators.py

```diff
@@ -66,6 +66,15 @@ class UserValidator(Schema):
     username = fields.String(required=True)
     password = fields.String(required=True)
 
+class UserPatchValidator(Schema):
+    """Validator for patching existing users
+    """
```

Comment:
Are any of these fields always required on a patch? If so, can modify the corresponding fields to be required as likely want to restrict the ways that a user record can be modified.